### DISCUSSION TITLE
Xdrip + Libre3

### DISCUSSION THREAD

**Discussion by Kustomwireless** [NightscoutFoundation/xDrip#3536] [Q&A]
Can someone explain why Xdrip + is not able to work direct with Libre 3 as some of the other Apps do such as Juggluco??
 Some back round   We have been using Xdrip + for about 8 years with US Libre and Blucon also bubble.

For Now We are using Juggluco to send data to Xdrip + and it seems to be working pretty well. With one exception and that is the sensors seem to stop working 2 to 3 days early consistently don't know if there is a connection or not!
 Just looking to simplify  things for my wife. Is there any other way accomplish  what we are doing?? I also follow her with Xdrip.

**Comment by PhilTU** [comment]
I think a simple answer is "nobody has done the work".

Juggluco uses a lot of vendor code I think (you have to have Librelink software either installed or present as an apk in order for Juggluco to install and work with a Libre 2) and the developer has done a lot of work in understanding how to transfer BLE connection between devices / apps on a running sensor.

A wider question is "why does xDrip+ not support all versions of Freestyle Libre" as the Libre 2 USA is also not supported (to name but one).
